src/main/java/org/broadinstitute/hellbender/engine/filters/OverclippedReadFilter.java

```diff
@@ -15,7 +15,7 @@
  *     From experience with sequencing of human DNA we have found cases of contamination by bacterial
  *     organisms; the symptoms of such contamination are a class of reads with only a small number
  *     of aligned bases and additionally many soft-clipped bases.  This filter is intended
- *     to remove such reads. Consecutive soft-clipped blocks are treated as a single block
+ *     to remove such reads. Consecutive soft-clipped blocks are treated as a single block.
```

Comment:
Done. And moved the sentence to a "Note" paragraph. Please, let me know if the example is right.